Resources/Prototypes/SS220/Entities/Mobs/NPCs/pets.yml

```
@@ -0,0 +1,34 @@
+- type: entity
+  name: Рокси
+  parent: MobSecDog
+  id: MobSecDogRoxi
+  description: Охраная собака — помощница офицеров, по поимки злодеев. Больно кусается, лучший офицер станции.
+  components:
+  - type: GhostRole
+    makeSentient: true
+    allowSpeech: true
+    allowMovement: true
+    name: "Рокси"
+    description: "Собака службы безопасности. Издевайтесь над офицерами, чтобы погладить их по голове и преследовать жестоких преступников по пятам"

```

```suggestion
    description: "Охранная собака. Оберегайте порядок, помогайте в поимке преступников."
```
Охранная собака звучит более нейтрально и более применимо к другим случаям использования. Остальную часть предложения вовсе не смог понять.